### ISSUE TITLE
sitewide 508-defect-4 [SCREENREADER]: Purely presentational content SHOULD NOT be announced to screen reader users 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18438]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I do not want to hear purely presentational content.

## Details

Two issues exist:
- On Safari/VO, the `<hr>` tags are still announced as a line break to the screen reader user.
- On Chrome/JAWs, the chevron next to the link within the frequently used VA forms is announced as a separate link next to the read more link.

## Acceptance Criteria

- [ ] Presentational content is not announced to the screen reader user

## Environment

* Mac, Safari/VO
* Windows, Chrome/JAWs

## Steps to Recreate

1. Enter `https://staging.va.gov/find-forms/` in Safari with VoiceOver turned on
2. Using the `up` and `down` keys, navigate through the form to the frequently used VA form cards
3. Confirm the `<hr>` is announced as a new line

1. Enter `https://staging.va.gov/find-forms/` in Chrome with JAWs turned on
2. Using the `up` and `down` keys, navigate to the "Read more >" link in the frequently used VA form cards
3. Confirm JAWs announces "Link, Read more..." first, and then upon a second `down` key, it announces "Link" again on the chevron.
4. Using the `up` and `down` keys, navigate to the pagination
5. Confirm JAWS announces "Single left pointing angle quotation mark" upon hovering over the chevron

## Proposed Solution (if known)

Unsure why Safari/VO is still announcing the `<hr>` as it has `aria-hidden="true"`.
As for JAWs, simply adding `aria-hidden="true"` to the `<i>` did the trick.

## Screenshots or Trace Logs

<img width="782" alt="Screen Shot 2021-01-12 at 5 09 42 PM" src="https://user-images.githubusercontent.com/14154792/104380574-381dd080-54f9-11eb-9b33-02cdd91e429a.png">
<img width="673" alt="Screen Shot 2021-01-12 at 5 10 34 PM" src="https://user-images.githubusercontent.com/14154792/104380575-381dd080-54f9-11eb-91cb-e97aedfea1b2.png">
<img width="1176" alt="Screen Shot 2021-01-12 at 5 23 22 PM" src="https://user-images.githubusercontent.com/14154792/104381790-07d73180-54fb-11eb-8a04-d1c0aac106bb.png">

**Comment by MarciMcGuireGCIO** [comment]
Moving all outstanding Find a VA Form 508 issues into a single Epic to clean up backlog.

**Comment by zacharymorel** [comment]
@joshkimux for the Pagination Chevron, have you made a 508 ticket for the design library? 
I can update the other issues here, but that one I might meet resistance. 
I can always ask if they'd be okay with using an icon verse the pseudo element `::after` and `::before`

**Comment by zacharymorel** [comment]
Also, the `<hr />` under the 3 cards there have `aira-hidden="true"` and I'm still seeing the issue on Safari/VO.

![Screen Shot 2021-04-09 at 3 01 53 PM](https://user-images.githubusercontent.com/26075258/114228576-a0d3e800-9944-11eb-8915-1fd1c0261264.png)

**Comment by joshkimux** [comment]
@zacharymorel If `role=presentation` and `aria-hidden="true"` both didn't work, there's nothing much more we can do here (other than to remove the `hr` completely, which I think would make for a cleaner design).

I'll make a ticket regarding the pseudo elements separate to this one!

Closing this out 😄

**Comment by MarciMcGuireGCIO** [comment]
Thank you, @zacharymorel and @joshkimux!
